scripts/vet.sh

```diff
@@ -90,7 +90,7 @@ for MOD_FILE in $(find . -name 'go.mod'); do
   go mod tidy -compat=1.21
   git status --porcelain 2>&1 | fail_on_output || \
     (git status; git --no-pager diff; exit 1)
- 
```

Comment:
I already fixed this in https://github.com/grpc/grpc-go/pull/7420, when it complained on my import. I didn't see the failure on `scripts/vet-proto.sh` during my import.